app/components/channel_list/categories/categories.tsx

```diff
@@ -44,6 +48,13 @@ const Categories = (props: Props) => {
             style={styles.flex}
             showsHorizontalScrollIndicator={false}
             showsVerticalScrollIndicator={false}
+            keyExtractor={extractKey}
+            removeClippedSubviews={true}
+            initialNumToRender={5}
+            windowSize={15}
+            updateCellsBatchingPeriod={10}
+            maxToRenderPerBatch={5}
+            viewabilityConfig={List.VISIBILITY_CONFIG_DEFAULTS}
```

Comment:
why do we care about the visibility of items in this list?

Reply:
Same as above - will remove and add later when needed.